### PR TITLE
docs: update `getting-started`

### PR DESCRIPTION
**Pull request by manchan4869**
Use `code-group` to provide tutorials for package managers

### CODE REVIEW

File: docs/guide/getting-started.md [hunk 1]
```diff
@@ -71,10 +95,21 @@ Add some scripts to `package.json`.
...
+```sh [npm]
+$ npm run docs:dev
+```
+
+```sh [yarn]
 $ yarn docs:dev
 ```
 
+```sh [pnpm]
+$ pnpm docs:dev
```
Comment (kiaking) [inline docs/guide/getting-started.md:109]: I think it would be safer to add `run` argument here?

```suggestion
$ pnpm run docs:dev
```

**Review by kiaking — COMMENTED** [review]
Thanks a lot! Added a comment 🙏

File: docs/guide/getting-started.md (hunk 1 above)
Comment (manchan4869) [reply docs/guide/getting-started.md]: Yeah,it would be safer

Comment (kiaking) [reply docs/guide/getting-started.md]: 👍

**Pull request merged**